src/dune/dune_file.ml

```diff
@@ -577,18 +472,51 @@ module Buildable = struct
     }
 
   let decode ~since_c ~allow_re_export =
+    let use_foreign =
+      Dune_lang.Syntax.deprecated_in Stanza.syntax (2, 0)
+        ~extra_info:"Use the (foreign_stubs ...) stanza instead."
```

Comment:
s/stanza/field/

"stanza" is for toplevel things

Reply:
Done